packages/benchmark/src/benchmark.js

```diff
@@ -0,0 +1,38 @@
+const performance = { now: () => Date.now() };
```

Comment:
I though `performance.now()` was microseconds. Sorry for the confusion. Now that I understand it is nanoseconds, my corresponding suggestion would be

```suggestion
const performance = { now: () => Date.now() * 1_000_000 };
```

to normalize to nanoseconds. But from your comments, it sounds like you agree we should normalize to nanoseconds, and that you're doing this somewhere. Where?

Reply:
Thanks for reminding.